src/app/zap-templates/zcl/data-model/silabs/ha.xml

```diff
@@ -2068,14 +2068,18 @@ limitations under the License.
         This message is used to return household appliance current alerts.
       </description>
       <arg name="alertsCount" type="AlertCount"/>
+      <!--
       <arg name="alertStructures" type="AlertStructure" array="true"/>
+      -->
     </command>
     <command source="server" code="0x01" name="AlertsNotification" optional="false">
       <description>
         This message is used to notify the current modification of warning and/or fault conditions.
       </description>
       <arg name="alertsCount" type="AlertCount"/>
+      <!--
       <arg name="alertStructures" type="AlertStructure" array="true"/>
+      -->
```

Comment:
Again, why is this being commented out?

Reply:
https://github.com/project-chip/connectedhomeip/pull/10159#issuecomment-934787625